Interfaces/API/Dirac.py

```diff
@@ -75,6 +81,39 @@ def __init__( self, withRepo = False, repoLocation = '', useCertificates = False
 
     self.__clients = None
 
+  def __checkFileArgument( self, fnList, prefix = None, single = False ):
```

Comment:
The following checks are to be done where necessary:
isinstance( value, ( int, long ) ) for all the ints
isinstance( value, ( int, long, float ) ) for all the numbers
isinstance( value, basestring ) for all the strings
isinstance( value, float ) 
isinstance( value, list )
isinstance( value, dict )
isinstance( value, set )
isinstance( value, tuple )